std/experimental/ndslice/slice.d

```diff
@@ -2934,7 +2941,7 @@ unittest
 
 private enum isSlicePointer(T) = isPointer!T || is(T : PtrShell!R, R);
 
-private struct LikePtr {}
+package struct LikePtr {}
```

Comment:
This is clear way and for internal use only. User defined ranges may brake introspection for pointer like behavior